### DISCUSSION TITLE
what is the meaning of `dwell` in `stops_visited`

### DISCUSSION THREAD

**Discussion by jlstpaul** [TIDES-transit/TIDES#153] [General]
What is the specific meaning of the `dwell` field in `stops_visited`.   

Is it always equal to `actual_departure_time` - `actual_arrival_time`?  If so, do we just have the field for convenience?   

Or is the meaning of `dwell` variable depending on the needs of the data creator?

**Comment by gabriel-korbato** [comment]
Dwell time could be calculated based on door opening and closing times or based on arrival and departure events, which could themselves come from doors or geofences.  It could be up to the user to calculate dwell times based on application, and the field isn't necessary.
